mobile_verifier/src/cli/server.rs

```diff
@@ -82,11 +82,26 @@ impl Cmd {
         .create()
         .await?;
 
+        // Seniority updates
+        let (seniority_updates, mut seniority_updates_server) = file_sink::FileSinkBuilder::new(
+            FileType::SeniorityUpdate,
+            store_base_path,
+            concat!(env!("CARGO_PKG_NAME"), "_seniority_update"),
+            shutdown_listener.clone(),
+        )
+        .deposits(Some(file_upload_tx.clone()))
+        .auto_commit(false)
+        .roll_time(Duration::minutes(15))
```

Comment:
Is 15 mins a reasonable duration taking into account the volume ? Seems high but I have no idea re expected volumes

Reply:
the volume of seniority_updates should actually be relatively low, we should only output this when the seniority_ts changes, which can be caused by a new coverage_object or going 72 hours with no heartbeats.